runtime/service/npud/core/DBus.cc

```diff
@@ -155,5 +156,81 @@ gboolean DBus::on_handle_request_destroy(NpudCore *object, GDBusMethodInvocation
   return TRUE;
 }
 
+gboolean DBus::on_handle_request_set_data(NpudCore *object, GDBusMethodInvocation *invocation,
+                                          guint64 arg_ctx, guint arg_rq_handle,
+                                          GVariant *arg_input_buffers, GVariant *arg_output_buffers)
+{
+  VERBOSE(DBus) << "on_handle_request_set_data with " << arg_ctx << ", " << arg_rq_handle
+                << std::endl;
+  GVariantIter *iter = NULL;
+  InputBuffers inBufs;
+  OutputBuffers outBufs;
+  gint type;
```

Comment:
Isn't it better to use `gint32`? (https://docs.gtk.org/glib/gvariant-format-strings.html#numeric-types)